docs/version0.32.md

```diff
@@ -58,6 +59,12 @@ For compatibility, the following OpenJDK HotSpot options are now supported by Op
 
 You can now use the `SharedClassStatistics` API to get the name, path, and directory of a shared classes cache. Depending on the operating system, you can also get the number of attached VMs for a non-persistent cache. This information is available through the following new methods: `cacheDir()`, `cacheName()`, `cachePath()`, and `numberAttached()`. For more information, see the API documentation.
 
+### Support for OpenSSL 3.0.x
+
+OpenSSL 3.0.x is supported but on Linux only. OpenSSL is enabled by default for the CBC, ChaCha20, ChaCha20-Poly1305Digest, GCM, and RSA cryptographic algorithms.
```

Comment:
Could use a link to the detailed info on OpenSSL too, eg "For more information about OpenSSL support, see [Cryptographic operations](introduction.md#cryptographic-operations)".